endpoints/calendar/index.js

```diff
@@ -43,7 +43,7 @@ const lookupHolidays = (yearStr, monthStr, dayStr) => new Promise((resolve, reje
       return current.date.toISOString().startsWith(`${year}-${month}-${day}`)
     })
     const results = holiday || {
-      date: `${year}-${month}-${day}T00:00:00.000Z`,
+      date: new Date(`${year}-${month}-${day}T00:00:00.000Z`),
```

Comment:
The `fridagar.getHolidays` function returns a response where the date param is a JS `Date` object.
So if we want to use moment dates we'd have to map through the responses and convert them, which I think is a bit out of scope for this PR.
On that note, IMO this endpoint  currently doesn't really need `moment` since its not manipulating the dates in any significant way.